dev-docs/source/howto/dbimports.rst

```diff
@@ -0,0 +1,18 @@
+Database Imports
+================
+
+Downloading Database Dump Files
+-------------------------------
+
+This command expects your private SSH key to be found in the ~/.ssh
+folder, in a file with the name datapunt.key (chmod 600)::
+
+    cp -i ~/.ssh/id_rsa.key ~/.ssh/datapunt.key
+
+Then download the data in the docker-based database::
+
+    docker-compose exec database update-db.sh dataservices
+
+To import the bag database do the following::
```

Comment:
This can be removed. The bag_v11 is not served through DSO-API anymore.

Reply:
Good point, removed completely (even from `docker-compose.yml` and others) in a separate commit